code/datums/components/food/edible.dm

```diff
@@ -498,26 +522,58 @@ Behavior that's still missing from this component that original food items had t
 		else if(foodtypes & H.dna.species.liked_food)
 			food_taste_reaction = FOOD_LIKED
 
-	if(HAS_TRAIT(parent, TRAIT_FOOD_SILVER)) // it's not real food
-		food_taste_reaction = isjellyperson(H) ? FOOD_LIKED : FOOD_TOXIC
+	if(HAS_TRAIT(parent, TRAIT_FOOD_SILVER) && !isjellyperson(H)) // it's not real food
+		food_taste_reaction = FOOD_TOXIC
 
-	switch(food_taste_reaction)
-		if(FOOD_TOXIC)
-			to_chat(H,span_warning("What the hell was that thing?!"))
-			H.adjust_disgust(25 + 30 * fraction)
-			H.add_mood_event("toxic_food", /datum/mood_event/disgusting_food)
-		if(FOOD_DISLIKED)
-			to_chat(H,span_notice("That didn't taste very good..."))
-			H.adjust_disgust(11 + 15 * fraction)
-			H.add_mood_event("gross_food", /datum/mood_event/gross_food)
-		if(FOOD_LIKED)
-			to_chat(H,span_notice("I love this taste!"))
-			H.adjust_disgust(-5 + -2.5 * fraction)
-			H.add_mood_event("fav_food", /datum/mood_event/favorite_food)
-			if(istype(parent, /obj/item/food))
-				var/obj/item/food/memorable_food = parent
-				if(memorable_food.venue_value >= FOOD_PRICE_EXOTIC)
-					H.add_mob_memory(/datum/memory/good_food, food = parent)
+	if(food_taste_reaction == FOOD_TOXIC)
+		to_chat(H,span_warning("What the hell was that thing?!"))
+		H.adjust_disgust(25 + 30 * fraction)
+		H.add_mood_event("toxic_food", /datum/mood_event/disgusting_food)
+		return
+
+	if(!istype(parent, /obj/item/food))
+		return
```

Comment:
Same here

Reply:
This one checks for `venue_value` so it has to remain.